examples/sms/Makefile

```diff
@@ -3,10 +3,10 @@ COMPILER ?= sccz80
 CFLAGS = -vn
 
 ifeq ($(COMPILER),sdcc)
-   CFLAGS += -compiler=sdcc  -SO3 --max-allocs-per-node200000 --fsigned-char 
-all: test.sms libctest.sms  apktest.sms testsnd.sms
+   CFLAGS += -compiler=sdcc -SO3 --max-allocs-per-node200000 --fsigned-char
+all: test.sms libctest.sms pongmstr.sms pngmstr2.sms chicken.sms testsnd.sms libpsgtest.sms
```

Comment:
Included more working examples for SDCC. apktest does not work with SDCC, it fails to link because the apk routine is missing.

Reply:
I've just exported the app symbols for classic/sdcc so I'll add apktest once I've merged this pr.